src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You can specify a generic font family as a fallback, such as 'serif', 'sans-serif', or 'monospace'.